### ISSUE TITLE
All translations need updates

### ISSUE THREAD

**Issue by erciccione** [monero-project/monero-gui#1116]
After #1076 all translations of the GUI need to be updated. Some are already in progress, but many other need to be checked and updated before the next release. 

These are the languages that aren't in porgress and need to be updated. I pinged in this issue all contributors who helped in past with the relative language, hoping they can contribute this time as well.

- [x] Croatian
- [x] German: @leonklingele @Schnoffel
- [x] Hebrew: @jonahar
- [x] Danish: @cryptobench
- [x] Portoguese Brazilian: @fabionitto @moneroj
- [x] Portuguese
- [x] Dutch: @potatored @fridzema @ProkhorZ
- [ ] Esperanto: @Keksoj
- [ ] Indonesian: ?
- [x] Japanese: @kenshi84
- [ ] Korean: @isaacdigs
- [x] Romanian: @mandrill-pie
- [x] Russian: @cheebeez @LvMsterfild

If you start working on a translation, please communicate it in the comments (if you have it, with the link to the WIP repository), so i can thick the relative language and consider it 'in progress'. Our tracking platform, [Taiga](https://taiga.getmonero.org/project/erciccione-monero-localization/backlog), is costantly updated and a great place to check what languages are in progress and who is working on them.

If you need help/infos ask down here, [PM me on reddit](https://www.reddit.com/message/compose/?to=ErCiccione) or come chat on `#monero-translations`

**Comment by ordtrogen** [comment]
Just mentioning I'm hard at work on the Swedish one. As soon as I have a PR I'll make it known and anyone'll be welcome to comment.

**Comment by ProkhorZ** [comment]
OK, thanks for the reminder. I started updating the Dutch translation. It's ~100 new or edited strings.

@potatored Thanks for correcting some bad typos. If you have time to contribute translations, could you work on the Monero website?
@fridzema I see you're busy coding. Great!

**Comment by netrik182** [comment]
The strings marked with ``<translation type="unfinished">`` are ready to be translated or should we expect any changes in the english text before the release?

**Comment by erciccione** [comment]
@netrik182 they are ready to be translated, just take off the `type="unfinished">` part and put your translation between `<translation></translation>`

**Comment by netrik182** [comment]
@erciccione ok, thanks!

**Comment by ProkhorZ** [comment]
@netrik182 If you use Qt Linguist, it will take you automatically to the new and updated strings, and it will remove the 'unfinished' tag for you. It has other useful features too, like copying translations for repeated strings. Annoyingly, there's no official distribution other than the whole Qt package, but the [download from Softpedia](http://www.softpedia.com/get/Others/Home-Education/Qt-Linguist.shtml) will probably be safe.

**Comment by netrik182** [comment]
@ProkhorZ I'll take a look. Thank you

**Comment by mandrill-pie** [comment]
Almost done with the Romanian translation. But here's a bit I'm having difficulties understanding:
```
Payment ID
A unique user name used in the address book.
It is not a transfer of information sent during the transfer.
```
What does that actually mean? Any other way to phrase it to help me translate it?

**Comment by mandrill-pie** [comment]
Also, what's the next step when I'm done? A PR like last time?

**Comment by Omgthehorror** [comment]
I was wondering about that same passage @mandrill-pie. The last sentence especially doesn't sound good in english either.

**Comment by LvMsterfild** [comment]
Perhaps, this is more correct link (
https://github.com/monero-project/monero-gui/pull/1076/files):
https://github.com/jonathancross/monero-gui/blob/84d363999139f8fb25e3a58d65508ec4c3592d24/translations/monero-core_ru.ts

2018-02-20 1:53 GMT+02:00 Omgthehorror <notifications@github.com>:

**Comment by mandrill-pie** [comment]
@erciccione Romanian updated. I've opened a pull request #1125

**Comment by erciccione** [comment]
@LvMsterfild that PR was merged into the code, no need to link it directly

@mandrill-pie thanks! i'm gonna review it

**Comment by takuto-h** [comment]
Hi, I've created a PR to update Japanese translation: #1132

**Comment by netrik182** [comment]
@erciccione Just created PR for Brazilian Portuguese #1136

**Comment by cryptobench** [comment]
PR submitted for danish translation #1142

**Comment by jonathancross** [comment]
@mandrill-pie You are correct, this is bad English.

Should be something like:

    Payment ID
    An arbitrary, unique identifier used to distinguish payments.
    The ID can be either 64 or 16 hexadecimal characters long.
    If 16, it will be encrypted, if 64, it will be visible on the blockchain.
    An ID can be combined with an address to form an Integrated Address.

Maybe best to fix the english and update all languages separately from this?

**Comment by Keksoj** [comment]
Hi, I'm the one who made the last translation fixes on the esperanto version. But if I check it at 
https://github.com/monero-project/monero-gui/blob/master/translations/monero-core_eo.ts 
I can't see any problem to it. Maybe I'm not looking good enough. Are they new lines to be added ?

Oh, okay, I just realised there's a bunch of <translation type="unfinished"></translation>
So, do I edit it, create a pull request and so on ? I'll have @erciccione help me for the details since I'm a bit of a noob. 
You can count on me anyway !

**Comment by pazos** [comment]
@erciccione: can we switch translation upgrades to a platform like transifex or  weblate? I think it would help contributors to get the work done.

**Comment by erciccione** [comment]
@pazos That's already in process. After the new release is out we will start working on the web platform. In our case, Pootle

**Comment by erciccione** [comment]
[The new releaseis out!](https://github.com/monero-project/monero-gui/releases/tag/v0.12.0.0)
Thanks everybody :)
